SDK_Guidelines.md

```diff
@@ -0,0 +1,16 @@
+# Process and Guidelines for Third Party Tools
+
+The KBase SDK is intended to lower the barrier to integrating analysis tools as KBase Methods and enable the broader community to more easily contribute to KBase. Tools submitted for addition  to KBase will go through a standard release process (Development, Beta and Production).
+
+## Tool states:
+1.	Development: Methods in the “Development” state will be limited in visibility. They will be visible in an publicly accessible environment but won’t run against the production data stores and can’t be used in production Narratives (i.e., the method will not be visible at narrative.kbase.us).
+2.	Beta: Methods in “Beta” will be visible to all users in the production Narrative Interface (narrative.kbase.us), if they chose to enable “Beta” mode in their Narrative.
+3.	Production: Methods in the Production state will be visible to all.  
+
+In order to transition a method from Development to Beta and, finally, Production, the Method must undergo a review by KBase staff. This review is intended to ensure that the Method is meeting a minimal set of guidelines and standards. Here are the general criteria that will be used when reviewing new methods which have been requested to be placed into a beta and production state.
+
+## Criteria:
+1.	All components of the tool are licensed for unrestricted open source use--for example, they cannot be “Free for academic use (but commercial users must pay to use it)”.
+2.	Documentation for Methods is clear and includes any appropriate references (e.g., published papers on a tool or method) [Link to Nomi’s page].
```

Comment:
https://github.com/kbase/project_guides/blob/master/Method_man_page.md
Is there a convention for using relative rather than absolute paths in these links?